packages/nc-gui-v2/components/dashboard/TreeView.vue

```diff
@@ -32,13 +30,9 @@ const tablesById = $computed<Record<string, TableType>>(() =>
 )
 
 const showTableList = ref(true)
-
 const tableCreateDlg = ref(false)
-
 const menuRef = $ref<HTMLLIElement>()
-
 let key = $ref(0)
-
 let sortable: Sortable
```

Comment:
Mh... I think leaving the spaces is better, otherwise the code becomes unreadable if there is no space between variables at all.

Reply:
Make sense,  will add them back :+1: